tools/render-test/slang-support.cpp

```diff
@@ -100,6 +100,7 @@ struct SlangShaderCompilerWrapper : public ShaderCompiler
                 (int)rawTypeNames.Count(),
                 rawTypeNames.Buffer());
 			int compileErr = spCompile(slangRequest);
+            spSetLineDirectiveMode(slangRequest, SLANG_LINE_DIRECTIVE_MODE_NONE);
```

Comment:
I assume this is debugging code; we probably don't want to leave it this way long-term.